C7GameData/AnimatedEffect.cs

```diff
@@ -0,0 +1,14 @@
+// My intention is that eventually AnimatedEffect will be a class with some useful stuff in it. Right now it's just an enum that lets the engine
+// communicate to the UI which Civ 3 special effect it wants to trigger. The UI must provide the file names for the animations, unfortunately they
+// can't be guessed based on the enum names (like MapUnit.AnimatedAction) because the files aren't all stored in predictable folder locations.
+namespace C7GameData
+{
+public enum AnimatedEffect {
+	Hit,
+	Hit2,
+	Hit3,
+	Hit5,
```

Comment:
Potentially silly question: What happened to Hit4?

More serious comment: It might be good to have a comment explaining why there are Hit, Hit2, Hit3, and Hit5.  This is probably documented on CFC somewhere?  But as someone who doesn't know the nitty-gritty on this, what's likely to happen is I'll see one of the code references using Hit5, dig down here, and still not really know why it called that one instead of one of the others.

Reply:
You'd have to ask someone at Firaxis what happened to Hit4. Those names come from the animation files in civ3PTW/Art/Animations/Trajectory. I wasn't sure how we should go about letting the engine tell the UI which effect animation to trigger, so to start I did the lazy thing and put an enum with one value for each Civ 3 animation file. Eventually we'll probably want a more abstract specification, i.e. the engine will just say to spawn a "hit" animation and let the UI pick one randomly from the Civ 3 ones. The hard part is that has to be made extensible. We don't want to assume the Civ 3 animations are available, and modders might want to add new effect animations or pick one non-randomly, etc.

Reply:
Interesting, that's a directory I'd never studied.  Looking at the files in FLICster, I'm not sure if "random" would be right though.  For example, hit.flc is 20 frames with a time of 833, and hit2.flc is 18 frames with a time of 692 (milliseconds?).

The comment helps, and we're probably good for now... it seems to be working.  But we might need to not only make it extensible, but see if we can figure out why there are 4 hit INI/FLC options... maybe different ones are used in different cases.